chart/flux/templates/deployment.yaml

```diff
@@ -148,6 +148,7 @@ spec:
 {{ toYaml .Values.extraEnvs | indent 10 }}
           {{- end }}
           args:
+          - --log-format={{ .Values.logFormat }}
```

Comment:
This should be set only if `.Values.logFormat` exists otherwise this introduces a breaking change.